assets/state-mig-manager/0400_configmap.yaml

```diff
@@ -83,6 +83,13 @@ data:
           mig-devices:
             "1g.20gb": 4
 
+      # B200
+      all-1g.23gb:
```

Comment:
I don't see configs for `1g.23gb+me` or `1g.45gb`

Reply:
Ahh yes, thanks for catching that :)